src/main/resources/templates/recall-summary.html

```diff
@@ -1012,14 +1020,15 @@
 <table class="c57">
   <tbody>
   <tr class="c49">
-    <td class="c39" colspan="1" rowspan="1"><p class="c0">
-      <p class="c24"><span class="c1">VULNERABILITY ISSUES: <span th:text="${vulnerabilityDiversityDetail}"></span></p>
+    <td class="c39" colspan="1" rowspan="1"><p class="c0 c24">
+        <span class="c1">VULNERABILITY ISSUES: </span>
+        <pre class="c24" th:text="${vulnerabilityDiversityDetail}"/>
```

Comment:
Just another example where I wouldn't think that we need to copy `c24` from the outer para onto the pre?

Reply:
A fair amount of this was trial and error. And once I got it working I stopped changing stuff, so there could be some unnecessary markup, but it all needs rewriting at some point. However, `pre` doesnt take any styling from parent elements so you need to add all the classes explicitly.